pkg/api/handlers/k8ssettings.go

```diff
@@ -0,0 +1,143 @@
+package handlers
+
+import (
+	"github.com/obot-platform/obot/apiclient/types"
+	"github.com/obot-platform/obot/pkg/api"
+	v1 "github.com/obot-platform/obot/pkg/storage/apis/obot.obot.ai/v1"
+	"github.com/obot-platform/obot/pkg/system"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/yaml"
+)
+
+type K8sSettingsHandler struct{}
+
+func NewK8sSettingsHandler() *K8sSettingsHandler {
+	return &K8sSettingsHandler{}
+}
+
+func (h *K8sSettingsHandler) Get(req api.Context) error {
+	var settings v1.K8sSettings
+	if err := req.Storage.Get(req.Context(), client.ObjectKey{
+		Namespace: req.Namespace(),
+		Name:      system.K8sSettingsName,
+	}, &settings); err != nil {
+		return err
+	}
+
+	converted, err := convertK8sSettings(settings)
+	if err != nil {
+		return err
+	}
+
+	return req.Write(converted)
+}
+
+func (h *K8sSettingsHandler) Update(req api.Context) error {
+	var input types.K8sSettings
+	if err := req.Read(&input); err != nil {
+		return err
+	}
+
+	// Retry loop to handle conflicts
+	var settings v1.K8sSettings
+	for i := 0; i < 5; i++ {
```

Comment:
I'm not 100% sure why I was experiencing 409s here to begin with, but the loop fixed it. If there is a better fix or something obviously wrong that was causing the 409s, please let me know.

Reply:
I think this was actually caused by a bug in Insomnia. It seems to have been firing off two (or more) requests in quick succession. I don't get 409s when I just use curl. So this seems to be a non-issue. I'm just gonna remove the loop.